samples/cpp/peopledetect.cpp

```diff
@@ -0,0 +1,115 @@
+#include <opencv2/objdetect.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+#include <iostream>
+
+using namespace cv;
+using namespace std;
+
+
+static void help()
+{
+    cout << "\nThis program demonstrates the use of the HoG descriptor using\n"
+            "  HOGDescriptor::hog.setSVMDetector(HOGDescriptor::getDefaultPeopleDetector());\n"
+            "Usage:\n"
+            "./peopledetect --i=<image_filename> | --d=<image_directory>\n\n"
+            "During execution:\n\tHit q or ESC key to quit.\n"
+            "\tUsing OpenCV version " << CV_VERSION << "\n" << endl;
+}
+
+const char* keys =
+{
+    "{help h||}"
+    "{image i| ../data/basketball2.png|input image name}"
+    "{directory d||images directory}"
+};
+
+void detectAndDraw(const HOGDescriptor &hog, Mat &img)
```

Comment:
Add `static` here or add prototype for this function to avoid compiler warning:

>opencv/samples/cpp/peopledetect.cpp:27:6: warning: no previous prototype for function 'detectAndDraw' [-Wmissing-prototypes]